tests/rptest/scale_tests/audit_log_test.py

```diff
@@ -393,24 +393,26 @@ def test_audit_log(self):
         self._enable_auditing()
         audit_enabled_results = self._run_repeater(topic_names, scale)
 
-        # Assert that there is no more then a x% difference in produce and consume throughput
-        allowable_threshold = 10.0
-
         def pct_difference(a, b):
             return (abs(a - b) / abs((a + b) / 2)) * 100
 
+        def within_threshold(a, b):
+            # Assert that there is no more then a x% difference in observed results
+            allowable_threshold = 10.0
+            # Ignore if a is worse then b
+            return a > b if pct_difference(a,
+                                           b) <= allowable_threshold else True
```

Comment:
This looks backwards to me. Shouldn't it be something like:

```suggestion
            return a < b or pct_difference(a, b) <= allowable_threshold
```

As in, either throughput `b` beat the baseline throughput `a` OR we're within acceptable bounds. 

I've been screwing up boolean algebra all week though, so I apologize in advance if I'm out to lunch or misinterpreting the independent variables.

Reply:
I love how boolean algebra is like the basic cornerstone of programming and even after years of experience it confounds us! :)